modules/imgcodecs/src/grfmt_tiff.cpp

```diff
@@ -269,6 +273,8 @@ bool TiffDecoder::readHeader()
                     m_type = CV_MAKETYPE(CV_64F, photometric > 1 ? 3 : 1);
                     result = true;
                     break;
+            default:
+                throw std::runtime_error("Invalid bitsperpixel value read from TIFF header! Must be 1, 8, 16, 32 or 64.");
```

Comment:
Use `CV_Error()` for error reporting.

Reply:
Test data: https://github.com/opencv/opencv_extra/pull/538

Ok I'll change that